python/pyfury/_serialization.pyx

```diff
@@ -1644,29 +1656,143 @@ cdef class CollectionSerializer(Serializer):
     cpdef int16_t get_xtype_id(self):
         return -FuryType.LIST.value
 
+    cpdef int8_t write_header(self, Buffer buffer, value):
```

Comment:
`cdef` is enough here, it will be invoked in ColllectionSerializer only

Reply:
done